packages/rx-utils/src/__tests__/create-handler.utils.spec.ts

```diff
@@ -1,24 +1,42 @@
 import { createHandler } from '../create-handler.utils';
 import { Endomorphism, Lazy } from 'fp-ts/lib/function';
+import { constVoid } from 'fp-ts/es6/function';
+
+type Effect<A> = (a: A) => void;
```

Comment:
This is not an "Effect". If you need a type for void thunk you can use `interface VoidThunk<A> { (a: A): void; }`

Reply:
is it important for the `.spec` file?

Reply:
fixed